lib/SILOptimizer/Transforms/StackPromotion.cpp

```diff
@@ -300,10 +300,6 @@ public std::iterator<std::forward_iterator_tag, SILBasicBlock, ptrdiff_t> {
                          NonUnreachableBlockIter rhs) {
     return lhs.BaseIterator == rhs.BaseIterator;
   }
-  friend bool operator!=(NonUnreachableBlockIter lhs,
-                         NonUnreachableBlockIter rhs) {
-    return !(lhs == rhs);
-  }
```

Comment:
Please don't delete this; it saves us from having to write it later. Just mark it LLVM_ATTRIBUTE_UNUSED or whatever.